cassandra-scheduler/src/dist/conf/scheduler.yml

```diff
@@ -41,7 +41,7 @@ cassandra:
      incremental_backups: false
      auto_snapshot: false
      snapshot_before_compaction: false
-     enable_user_defined_functions: false
+     enable_user_defined_functions: ${CASSANDRA_ENABLE_USER_DEFINED_FUNCTIONS:-false}
```

Comment:
Can you please move this to a separate PR ? As this doesn't relate to coalescing strategy.